packages/iris-grid/src/IrisGridTableModelTemplate.ts

```diff
@@ -2054,11 +2062,11 @@ class IrisGridTableModelTemplate<
     }
   }
 
-  editValueForCell(x: ModelIndex, y: ModelIndex): string | null | undefined {
-    return this.textValueForCell(x, y);
+  editValueForCell(column: ModelIndex, row: ModelIndex): string {
+    return this.textValueForCell(column, row) as string;
```

Comment:
We should avoid casting whenever possible. In this case, it would be bad to return possibly `null` or `undefined` 

```suggestion
    return this.textValueForCell(column, row) ?? '';
```